tools/Dashboard/DevHome.Dashboard.UnitTest/DevHome.Dashboard.UnitTest.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
     <PackageReference Include="coverlet.collector" Version="3.1.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.4" />
```

Comment:
It seems like this is going into a lot of csproj's. Does it need to? If it does, can we put it in Common?